iso/Dockerfile

```diff
@@ -20,6 +20,10 @@ FROM boot2docker/boot2docker:1.12.3
 RUN echo "1.12.3" > $ROOTFS/etc/version
 ENV DOCKER_SHA a3bb88649bcc47775c29dab7d95709f4dda04d0e
 
+# Add the /etc/os-release file
+RUN sed -i '/^b2dVersion/,/^EOOS/d' /make_iso.sh
```

Comment:
So you don't want to make _make_iso.sh_ part of the project? Can you then at least a pointer to what you are doing there. For example a link to make_iso.sh in the boot2docker repo